routes/error-tracker.js

```diff
@@ -225,25 +234,28 @@ function getHandler(req, res) {
     },
     severity: severity,
   };
+  unminify.unminify(exception).then(function(unminifiedException) {
+    exception = params.m + '\n' + unminifiedException;
+  }, function(err) {
+    exception = params.m + '\n' + exception;
+    console.log(err);
+  });
   const entry = log.entry(metaData, event);
```

Comment:
This will never have an unminified stack trace, since the above promise will not have executed yet.